docs/user-guide/vwii/finalizing-setup.md

```diff
@@ -0,0 +1,31 @@
+# vWii Modding Guide {docsify-ignore-all}
+
+## Finalizing Setup
+
+Now that the required cIOSes and IOS80 patch have been applied, we are now going to finalize the setup.
+
+### Additional Homebrew Apps
+
+- **SaveGame Manager GX** allows you to backup and restore your saves and Miis, especially saves that can't usually be copied.
+- **Nintendont** is a virtual machine that allows you to play Gamecube games.
+- The **Open Shop Channel** is an appstore where you can download more apps.
+
+### Additional Homebrew Apps - What You Need
+
+- The latest release of [SaveGame Manager GX](https://hbb1.oscwii.org/api/v3/contents/SaveGame_Manager_GX/SaveGame_Manager_GX.zip)
+- The `loader.dol`, `meta.xml` and `icon.png` of [Nintendont](https://github.com/GaryOderNichts/Nintendont)
```

Comment:
Can't we just link the one on OSC? Gary's changes have been merged upstream afaik

Reply:
So is it the same one that fixed the black screen controller issue?

Reply:
Actually, im not sure anymore. I was told that it was merged upstream but i do not see any PR from Gary to the Nintendont repo.

Reply:
I suppose we could